src/common/vfs.h

```diff
@@ -113,6 +113,15 @@ class OCSYNC_EXPORT Vfs : public QObject
     };
     Q_ENUM(ConvertToPlaceholderResult)
 
+    enum UpdateMetadataType {
+        DatabaseMetadata = 0 >> 1,
```

Comment:
@mgallien all enum values are 0, so, could you explain what happens hear?

Reply:
fixed